toolbox/githubctl/main.go

```diff
@@ -45,14 +60,111 @@ func fastForward(repo, baseBranch, refSHA *string) error {
 	return githubClnt.FastForward(*repo, *baseBranch, *refSHA)
 }
 
+func createIstioReleaseTag() error {
+	assertNotEmpty("base_branch", baseBranch)
+	pickledDeps, err := githubClnt.GetFileContent(istioRepo, *baseBranch, istioDepsFile)
+	if err != nil {
+		return err
+	}
+	deps, err := u.DeserializeDepsFromString(pickledDeps)
+	if err != nil {
+		return err
+	}
+	releaseTag, err := githubClnt.GetFileContent(istioRepo, *baseBranch, releaseTagFile)
+	if err != nil {
+		return err
+	}
+	releaseTag = strings.Trim(releaseTag, "\n")
+	releaseMsg := "Istio Release " + releaseTag
+	for _, dep := range deps {
+		if err := githubClnt.CreateAnnotatedTag(
+			dep.RepoName, releaseTag, dep.LastStableSHA, releaseMsg); err != nil {
```

Comment:
Are we guaranteed to have dep.LastStableSHA in sync with the value in istio.VERSION ?